torchinductor/triton_ops/conv1x1.py

```diff
@@ -0,0 +1,144 @@
+import torch
+import triton
+
+import triton.language as tl
+
+
+class _conv1x1(torch.autograd.Function):
```

Comment:
Could we refactor this to share more code with the file above?

Reply:
Move to utils.py